### DISCUSSION TITLE
[Desktop] Need ability to change task tray icon size in preference menu

### DISCUSSION THREAD

**Discussion by sulbig** [wireapp/wire-desktop#5067] [Ideas]
Wire version: 3.3.2872
Wire for web version: 2018.10.15.0814
Operating system: Linux Mint 18.3 Cinnamon 64-bit

Issue?
The task tray icon on a UHD 4K2K resolution is super tiny. Need the ability to select different (larger) task tray icon sizes in the preference menu. Today, I must manually edit the the .png file names (e.g., tray@2x.png) in /opt/Wire/resources/app/js/menu/TrayHandler.js to achieve the desired results.

**Comment by kacperpaczos** [comment]
Right!
The icon is too small for me.
Linux Mint Cinammon 19
![obraz](https://user-images.githubusercontent.com/23406555/47320222-16527d80-d651-11e8-8306-cddd4bb770c5.png)

**Comment by bennycode** [comment]
@sulbig Can you please submit a PR with your enlarged tray icon? We can commit it to our codebase if it is useful for others.
